java/system-test/system-test-drivers/src/main/java/sleeper/systemtest/drivers/compaction/AwsCompactionDriver.java

```diff
@@ -98,19 +102,7 @@ public List<String> forceCreateJobsGetIds() {
     }
 
     public void invokeTasks(int expectedTasks, PollWithRetries poll) {
```

Comment:
We probably want to leave this driver as it was before, and we can wire the new behaviour in here as a separate operation later.

Reply:
Thanks, i've reverted this change